openedx_events/event_bus/avro/custom_serializers.py

```diff
@@ -149,11 +150,33 @@ def deserialize(data: str):
         return LibraryUsageLocatorV2.from_string(data)
 
 
+class UuidAvroSerializer(BaseCustomTypeAvroSerializer):
```

Comment:
What about:
```suggestion
class UUIDAvroSerializer(BaseCustomTypeAvroSerializer):
```

Reply:
I don't personally have a preference, but that seems inconsistent with `CcxCourseLocatorAvroSerializer` further up in this file.

Reply:
That makes more sense.